### ISSUE TITLE
chatwoot integration unable to determine webhook protocol when https is enabled and host is missing protocol

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2884]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.46.3](https://github.com/open-wa/wa-automate-nodejs/compare/4.46.2...4.46.3) (2022-09-14)

- fix #2884 [`#2884`](https://github.com/open-wa/wa-automate-nodejs/issues/2884)
- Release 4.46.3 [`94aa81e`](https://github.com/open-wa/wa-automate-nodejs/commit/94aa81ea3fc5ed6ebbdece009050377099575026)
- updated types-only package [`a89bfee`](https://github.com/open-wa/wa-automate-nodejs/commit/a89bfee7b38714d9cde8aca93c928a70c68c707c)
